src/main/kotlin/io/bazel/worker/PersistentWorker.kt

```diff
@@ -83,6 +84,7 @@ class PersistentWorker(
         blockable {
```

Comment:
unblockedContext on line 60 is unused, is that intentional?

Reply:
For now, yeah.